ci/gpu/build.sh

```diff
@@ -52,8 +52,8 @@ conda install "cudatoolkit=$CUDA_REL" \
               -c rapidsai-nightly
 
 # needed for asynccontextmanager in py36
-conda install -c conda-forge "async_generator" "automake" "libtool" \
-                              "cmake" "automake" "autoconf" "cython>=0.29.14,<3.0.0a0" \
+conda install -c conda-forge "async_generator" "autoconf" "automake" "libtool" \
```

Comment:
Please remove the `-c conda-forge` as channels are now set via a `.condarc` file.

Can any of these packages be pushed into the integration repo to avoid needing to pull them each time in this script?

Reply:
Thanks for the info. Can you please provide context as to where this is done? Where can we see this?

We can't use the integration packages here as they require `ucx-py` ( https://github.com/rapidsai/ucx-py/pull/548#issuecomment-644287361 )

Reply:
`.condarc` file is set in the gpuCI image here: https://github.com/rapidsai/gpuci-build-environment/blob/branch-0.15/rapidsai/devel.Dockerfile#L27-L47

Reply:
> Thanks for the info. Can you please provide context as to where this is done? Where can we see this?
> 
> We can't use the integration packages here as they require `ucx-py` ( [#548 (comment)](https://github.com/rapidsai/ucx-py/pull/548#issuecomment-644287361) )

We have been wanting to remove `ucx-py` from `rapids-build-env`. If this would help in this case then we should get a list of all of the repos impacted by that change and make it happen. If you agree to the removal can you file an issue with ops and we'll start the process.

Reply:
As per https://github.com/rapidsai/ucx-py/pull/570#issuecomment-669016867, I think we should just stop building UCX from source and rely on UCX conda package, that way we'll make sure we test it to some extent as well which doesn't happen today.

Reply:
Thanks Mike! My guess is we are probably ok (especially now that the list has been significantly reduced here by using the `ucx` package). Plus it will probably help us to track these in `ucx-py` as we migrate things upstream to help make sure we don't miss any dependencies in the process.